src/pages/EditorPage/EditorPanel.jsx

```diff
@@ -617,6 +627,20 @@ const EditorPanel = ({
                       widthExpand
                     />
                   )
+                    } else if (field === 'tags') {
+                      input = (
+                    <TagsSelect
+                      value={isMultiple ? union(...isMultiple) : value || []}
```

Comment:
Lets go ahead and change this confusing variable name to something else like `mixedValue`.

Reply:
I changed `isMultiple` to `multipleValues`, I was thinking about `mixedValues` but technically they are not mixed, it is the union function that mixes them.